graphics/fontforge/Portfile

```diff
@@ -93,67 +46,38 @@ depends_lib \
+                port:libspiro \
+                port:pango \
+                port:cairo
 
 configure.args  --mandir=${prefix}/share/man \
-                --enable-type3 \
-                --enable-devicetables \
-                --enable-gb12345 \
-                --without-python \
-                --without-freetype-bytecode \
-                --without-pango \
-                --without-cairo \
-                --x-includes=${prefix}/include \
-                --x-libraries=${prefix}/lib
+                --disable-python-scripting \
+                --disable-python-extension \
+                --without-x
 
 configure.ldflags-append \
                 -lintl
 
-# https://trac.macports.org/ticket/29840
-compiler.blacklist \
-                llvm-gcc-4.2
-
-set docdir      ${prefix}/share/doc/${name}
-destroot.args   docdir=${docdir}
-
-post-destroot {
-    xinstall -d ${destroot}${docdir}
-    xinstall -m 644 -W ${worksrcpath} \
-        AUTHORS INSTALL LICENSE README-Mac.html README-Unix.html README-unix \
-        ${destroot}${docdir}
-    # Fix permissions problem on this file
-    xinstall -m 0644 -W ${destroot}${applications_dir}/FontForge.app/Contents/Resources FontForge.icns \
-        ${destroot}${applications_dir}/FontForge.app/Contents/Resources
-}
-
-variant freetype_bytecode description {Enable support for bytecode interpreter} {
-    distfiles-append        freetype-${ftversion}${extract.suffix}:freetype
-    extract.only-append     freetype-${ftversion}${extract.suffix}
-    configure.args-delete   --without-freetype-bytecode
-    configure.args-append   --with-freetype-src=${workpath}/freetype-${ftversion}
-}
+#PYTHON_CFLAGS=`${PYTHON}-config --cflags`
+#PYTHON_LIBS=`${PYTHON}-config --ldflags`
 
 variant python26 conflicts python27 description {Enable Python support (Python 2.6)} {
-    configure.args-delete   --without-python
-    configure.args-append   --enable-pyextension \
-                            --with-python=${prefix}/bin/python2.6 \
-                            --with-python-headers=${frameworks_dir}/Python.framework/Versions/2.6/include/python2.6
+    configure.args-delete   --disable-python-scripting \
+                            --disable-python-extension
+    configure.env-append    PYTHON=${prefix}/bin/python2.6 \
+                            PYTHON_CFLAGS="-I${frameworks_dir}/Python.framework/Versions/2.6/include/python2.6 -I${frameworks_dir}/Python.framework/Versions/2.6/include/python2.6 -fno-strict-aliasing -fno-common -dynamic -pipe -Os -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes" \
+                            PYTHON_LIBS="-L${frameworks_dir}/Python.framework/Versions/2.6/lib/python2.6/config -lpython2.6 -ldl -framework CoreFoundation"
     destroot.args-append    PYTHON=${prefix}/bin/python2.6 PYTHONPREFIX=${frameworks_dir}/Python.framework/Versions/2.6
     depends_lib-append      port:python26
 }
 
 variant python27 conflicts python26 description {Enable Python support (Python 2.7)} {
```

Comment:
Any reason not to make python27 variant default?

Just curious, did you test whether python 3 works?

Reply:
Ha, ok. No, no reason. see above ;-)

Reply:
No, I haven't tried python 3.